validate-charts/entrypoint.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh -l
+
+echo "validating helm charts"
```

Comment:
Will this action work on the latest checkout code?

Reply:
Yes it mounts repo it is running in as working directory. 

see this step: https://github.com/hypertrace/attribute-service/runs/1641880227#step:7:93

Reply:
I see, I checked it here - https://github.com/hypertrace/attribute-service/runs/1641880227#step:7:93

Reply:
So, we are going to do the same for publishing action -` hypertrace/actions/helm-gcs-publish`?

Reply:
I think sharing an image and script and calling two different commands that the script recognizes could allow us to share most of this (i.e. `entrypoint.sh publish <other args>`, `entrypoint.sh validate <other args>`)

Reply:
updated this.